samples/gazell/gzll_ack_payload_device/README.rst

```diff
@@ -0,0 +1,22 @@
+.. _gzll_ack_payload_device:
+
+Gazell ACK Payload Device
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample shows basic Gazell communication and demonstrates how to send payloads and acknowledgments.
+It consists of two applications, one running on the device and one running on the host.
```

Comment:
Capitalization for device and host is not needed here?

Reply:
I think you are right. I will implement this change.